File: resoluciones/javascript/tareas2.js
//1)
function medidorDelMayor(a,b) {
	if (a < b) {
		console.log(a)
	}
	else if (a == b) {
		console.log(`tus dos parametros son iguales`)
	}
	else {
		console.log(b)
	}
}

console.log(medidorDelMayor(15,1))


//2)
let primerValor1 = Number(prompt(`deme su primer valor`))
let segundoValor1 = Number(prompt(`deme su segundo valor`))
let tercerValor1 = Number(prompt(`deme su tercer valor`))
function elMayor(A,B,C){
    if(A > B & A > C){
        console.log(A)
    }
    else if(B > A & B > C){
        console.log(B)
    
    }
    else if(C > B & C > A){
        console.log(C)
    
    }
}

console.log(elMayor(primerValor1,segundoValor1,tercerValor1))


//3)
let primerValor = Number(prompt(`deme su primer valor`))
let segundoValor = Number(prompt(`deme su segundo valor`))
let tercerValor = Number(prompt(`deme su tercer valor`))
function deMenorAMayor(A,B,C){
    while(A == B || B == C ||A == C){
        alert(`ingrese de nuevo sus datos`)
        A = Number(prompt(`deme su primer valor`))
        B = Number(prompt(`deme su segundo valor`))
        C = Number(prompt(`deme su tercer valor`))
    }
    if(A > B & B > C){
        console.log(C,B,A)

        }
    else if(A > C & C > B){
        console.log(B,C,A)
    }
    else if(B > A & A > C){
        console.log(C,A,B)

    }
    else if(B > C & C > A){
        console.log(A,C,B)

    }
    else if(C > B & B > A){
        console.log(A,B,C)

    }
    else if(C > A & A > B){
        console.log(B,A,C)

    }

}

console.log(deMenorAMayor(primerValor,segundoValor,tercerValor))

File: resoluciones/python/tareas2.py
#1)
primerValor = int(input("deme su primer valor"))
segundoValor = int(input("deme su segundo valor"))
def medidorDelMenor(a,b):
	if (a < b):
		return(a)
	elif (a == b):
		return("tus dos parametros son iguales")
	else:
		return(b)
		
print(medidorDelMenor(primerValor,segundoValor))

#2)
primerValor1 = int(input("deme su primer valor"))
segundoValor1 = int(input("deme su segundo valor"))
tercerValor1 = int(input("deme su tercer valor"))

def medidorDelMayor(A,B,C):
    if(A > B & A > C):
        return(A)
    elif(B > A & B > C):
        return(B)
    elif(C > B & C > A):
        return(C)

print(medidorDelMayor(primerValor1,segundoValor1,tercerValor1))

#3)
def deMenorAMayor(A,B,C):
    if(A > B & B > C):
        return(C,B,A)
    elif(A > C & C > B):
        return(B,C,A)
    elif(B > A & A > C):
        return(C,A,B)
    elif(B > C & C > A):
        return(A,C,B)
    elif(C > B & B > A):
        return(A,B,C)
    elif(C > A & A > B):
        return(B,A,C)

print(deMenorAMayor(primerValor1,segundoValor1,tercerValor1))